include/ebpf_result.h

```diff
@@ -112,6 +112,9 @@ extern "C"
 
         /// Operation timed out.
         EBPF_TIMEOUT,
+
+        /// Key is valid, but the object has been deleted
```

Comment:
```suggestion
        /// Key is valid, but the object has been deleted.
```
Use style consistent with rest of defines above

Reply:
fixed